lib/matplotlib/cbook.py

```diff
@@ -783,8 +783,12 @@ def is_sequence_of_strings(obj):
     """
     if not iterable(obj):
         return False
-    if is_string_like(obj):
-        return False
+    if is_string_like(obj) and not isinstance(obj, np.ndarray):
```

Comment:
But we can not import pandas directly in core matplotlib for both technical
and policy reasons.

On Tue, Apr 28, 2015, 11:31 Phillip Cloud <notifications@github.com> wrote:

> In lib/matplotlib/cbook.py
> <https://github.com/matplotlib/matplotlib/pull/4358#discussion_r29254616>:
>
> > @@ -783,8 +783,12 @@ def is_sequence_of_strings(obj):
> >      """
> >      if not iterable(obj):
> >          return False
> > -    if is_string_like(obj):
> > -        return False
> > +    if is_string_like(obj) and not isinstance(obj, np.ndarray):
>
> What does is_string_like(series of str) do? Does it loop through the
> array checking to see if each element is a str?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4358/files#r29254616>.
>